server/src/controllers/Events.js

```diff
@@ -54,7 +55,10 @@ module.exports = {
       const eventId = req.params.id;
       const file = req.file;
       if (file) {
-        await cloudinary.v2.uploader.destroy(eventId);
+        try {
+          await cloudinary.api.resource(eventId);
+          await cloudinary.v2.uploader.destroy(eventId);
+        } catch(error) {}
```

Comment:
It's better if send an error response here, presumably with a 500 response since something failed on the server side.